c/termination-recursive-malloc/chunk3_true-termination.c

```diff
@@ -0,0 +1,65 @@
+#include <stdlib.h>
+
+extern int __VERIFIER_nondet_int(void);
+
+void chunk(int **data) {
+	int *p0 = data[0];
+	int *p1 = data[1];
+	int *p2 = data[2];
+	int *p3 = data[3];
+	
+	if(*p0 >= *p3) 
+		return;
+	
+	(*p0)++;

```

Comment:
hm, actually there cannot be an overflow, because *p3 has to be bigger than *p0.